src/app/modules/example-feature/components/presentation/presentation.component.ts

```diff
@@ -1,10 +1,12 @@
 import { Component, OnInit } from '@angular/core';
+import { ExamplesService } from '../../state/examples.service';
 
 @Component({
   selector: 'app-presentation',
   templateUrl: './presentation.component.html',
   styleUrls: ['./presentation.component.scss'],
 })
 export class PresentationComponent implements OnInit {
+  constructor(private exampleService: ExamplesService) { this.exampleService.loadExamples().subscribe(s => console.log(s)); }
```

Comment:
Use:

```
constructor(private exampleService: ExamplesService) {
    this.exampleService.loadExamples().subscribe(s => console.log(s));
}
```

for formatting `.ts` files.

Reply:
I just updated this branch with master and now our linter throws an error on this kind of issues :) 

You can even view the errors on https://github.com/carbonaut-io/pwa-boilerplate/pull/14/files

Reply:
Done